sagemaker-fundamentals/create-training-job/create_training_job_vpc.ipynb

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
<p>I know from blog posts that <code>you</code>/second-person is preferred over <code>we</code>/first-person. You use <code>we</code>/<code>let's</code> a few times - those could all be changed to <code>you</code> if the notebook style is the same.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
going to look through the nb again and replace all instances of "we" by "you"